pytorch_toolkit/ote/ote/modules/builder.py

```diff
@@ -16,7 +16,7 @@
 
 from mmcv.utils import Registry
 
-from .registry import ARG_PARSERS, ARG_CONVERTERS, TRAINERS, EVALUATORS, EXPORTERS
+from .registry import ARG_PARSERS, ARG_CONVERTERS, TRAINERS, EVALUATORS, EXPORTERS, CONFIG_TRANSFORMERS
```

Comment:
Can we avoid using the word `TRANSFORMERS`? I am ok with compiler, converter, transformations, etc.

Reply:
Oh... ALexander, I tried to find the best name.

@Ilya-Krylov , what is your opinion?

Reply:
I think it is premature generalization. We don't have more than one NNCF transformer (there are likely to be more than one for other algos), but it is questionable to have something else than NNCF config transformer.

I would name it COMPRESSION_CONFIG_<something>

Reply:
@Ilya-Krylov , @AlexKoff88 , @druzhkov-paul :  
Inspired by your comments I significatly simplified my classes and removed premature generalization stuff in the classes, transforming arguments for compression.
Please, see my commit 73ffe6b.

Note that the main classes are renamed and refactored:
* NNCFConfigTransformer -> NNCFConfigGenerator
* ConfigTransformersEngine -> NNCFConfigTransformer

Also other classes are refactored
* _ConfigTransformersHandler simplified and merged into
   new class NNCFConfigTransformer
* registry CONFIG_TRANSFORMERS -> registry COMPRESSION

So, the main class in compression arguments transforming now is NNCFConfigTransformer, and it is accessible
through registry COMPRESSION.